src/pages/index.js

```diff
@@ -14,41 +16,59 @@ const profileEditBtn = document.querySelector("#profile-edit-btn");
 const profileTitleInput = document.querySelector("#profile-title-input");
 const profileSubtextInput = document.querySelector("#profile-subtext-input");
 const profileModalForm = document.querySelector("#profile-modal-form");
-const profileTitle = document.querySelector("#profile-title");
-const profileSubtext = document.querySelector("#profile-subtext");
 
 const addBtn = document.querySelector("#add-button");
+const avatarEditBtn = document.querySelector("#avatar-edit-btn");
 
-const newCardTitleInput = document.querySelector("#new-card-title-input");
-const newCardLinkInput = document.querySelector("#new-card-link-input");
 const newCardModalForm = document.querySelector("#new-card-modal-form");
+const avatarModalForm = document.querySelector("#avatar-modal-form");
 
 const cardListElement = document.querySelector(".cards__list");
 
 /**
- * Functions
+ * Variables
  */
 
-// Section Functionality
+const editFormValidator = new FormValidator(config, profileModalForm);
+const cardFormValidator = new FormValidator(config, newCardModalForm);
+const avatarFormValidator = new FormValidator(config, avatarModalForm);
+editFormValidator.enableValidation();
+cardFormValidator.enableValidation();
+avatarFormValidator.enableValidation();
 
 const renderCard = (cardData) => {
-  const card = new Card(cardData, "#card-template", handleImageClick);
+  const card = new Card(
+    cardData,
+    "#card-template",
+    handleImageClick,
+    handleDeleteClick,
+    handleLikeClick
+  );
   const cardElement = card.getView();
   section.addItem(cardElement);
 };
 
-const section = new Section(
-  { items: initialCards, renderer: renderCard },
-  cardListElement
-);
-section.renderItems();
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "03a6e48e-c1e2-40b5-8f26-406bbefbcb53",
+    "Content-Type": "application/json",
+  },
+});
 
-// Validation Functionality
+let section;
+api.getInitialCards().then((items) => {
+  section = new Section(
+    { items: items.reverse(), renderer: renderCard },
+    cardListElement
+  );
+  section.renderItems();
+});
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere**